protocols/upnp/src/behaviour.rs

```diff
@@ -245,10 +238,9 @@ where
     P: Provider + 'static,
 {
     /// Builds a new `UPnP` behaviour.
-    pub fn new(config: Config) -> Self {
+    pub fn new() -> Self {
```

Comment:
Do we need the `new` function when we have `default`? If we omit it, we could later add one that has a config parameter as a non-breaking change.

Reply:
yup, agreed. Updated, thanks Thomas